src/modules/Scraper/utils/downloader.ts

```diff
@@ -52,6 +52,15 @@ const removeBaseTag = async (page: Page) => {
     document.querySelector('base')?.remove();
   });
 };
+/*
+ * Some sites have a protection against headless browsers
+ * The only fact to put a user agent will bypass this protection
+ *
+ * Initially done for https://napoveda.seznam.cz/cz/sreality/pravidla-sreality/smluvni-podminky-sluzby-sreality.cz/smluvni-podminky-pro-vkladani-inzerce-do-databaze-serveru-sreality.ct-platne-13.12.2021
+ */
+const setCustomUserAgent = async (page: Page) => {
+  await page.setUserAgent('OpenTermsArchive/contribution-tool');
```

Comment:
Don't you think it would be safer to generate a unique and random name?

Reply:
I think for now it does not matter. We'll see if it generates problems on other documents and fine tune from there